cantools/database/can/c_source.py

```diff
@@ -417,11 +417,11 @@
  *
  * @return zero(0) on success or (-1) in case of nullptr argument.
  */
-int {database_name}_{message_name}_init(struct {database_name}_{message_name}_t *msg_p);
```

Comment:
since everything in the C code is in terms of raw values anyways, the concept of  initializing these objects to the init values is independent from  whether you do this using physical or raw values, these functions should not exhibit `_raw` suffixes. (they are basically the constructors of these structures...)

IOW, please revert this patch...

Reply:
Okay, I'll revert last changes